libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/initializeDashboardHandler/index.ts

```diff
@@ -163,14 +161,18 @@ function* loadExistingDashboard(
         call(resolveDashboardConfig, ctx, cmd),
         call(resolvePermissions, ctx, cmd),
         call(resolveEntitlements, ctx),
-        call(loadCatalog, ctx),
         call(loadDashboardAlerts, ctx),
         call(loadUser, ctx),
         call(loadDashboardList, ctx),
         call(loadAccessibleDashboardList, ctx),
         call(loadLegacyDashboards, ctx),
         call(loadDashboardPermissions, ctx),
     ]);
+    const catalog: PromiseFnReturnType<typeof loadCatalog> = yield call(
+        loadCatalog,
+        ctx,
+        config?.objectAvailability,
```

Comment:
objectAvailability is actually already part of cmd.payload.config available from start